### DISCUSSION TITLE
Issue with login to Poweradmin

### DISCUSSION THREAD

**Discussion by jpetiot** [poweradmin/poweradmin#447] [Q&A]
Hello,

Strange issue came up recently: I cannot login to Poweradmin anymore; everytime I do so I get a "Authentication failed!" message.
I checked the admin username and the password hash in the mysql database and they match exactly the admin username and the password hash of the password I am indicating on the login page.

"systemctl status pdns" does not report any issues.

Versions are PowerDNS Authoritative Server 4.1.14, poweradmin 2.1.7 and PHP 7.2.27

Obviously the poweradmin version is very old but it was working fine.

Actually was considering upgrading poweradmin but not sure if there is a recommended procedure to do so?

Any guidance would be welcome.

**Comment by edmondas** [comment]
@jpetiot check your http server logs.

The common steps for upgrading the system:
- Backup your database (always)
- Backup old files, replace everything with content from downloaded file
- From backup restore only inc/config.inc.php
- Update database structure (if needed)
